### DISCUSSION TITLE
cEOS : Ubuntu live server 20.04.3 : Increasing Limits

### DISCUSSION THREAD

**Discussion by dharmbhai** [srl-labs/containerlab#673] [Ideas] [answered]
it's possible that containers will set them back so we may have to keep setting them as the containers are booting
Add following file to /etc/sysctl.d/98-containerlab-inotify.conf
fs.inotify.max_user_instances = 32768
fs.inotify.max_user_watches = 524288

Then once your large cEOS nodes based topology is being deployed using containerlab, issue below cmd to increase the limits
sudo sysctl -p /etc/sysctl.d/98-containerlab-inotify.conf
fs.inotify.max_user_instances = 32768
fs.inotify.max_user_watches = 524288

**Comment by hellt** [comment]
This is node specific setting. I think users will have to set it if they need to not impose changes when they are not needed

**Comment by dharmbhai** [comment]
Sure @hellt : Totally agree: I just want to know if we can have this enhancement. 

Right now, I have to do it via CLI separately. But if there is a possibility to add an option to run a script which is provided as an input to clab.yml file, it would be awesome

**Comment by hellt** [comment, marked as answer]
Does an input to clab make it easier than calling the script like this?

setsysctl.sh && clab deploy ...

**Reply by dharmbhai** [reply]
Hi @hellt : Arista cEOS reset the Limits once they are coming up. So this has to be done post deploy
That said, I'll test your recommendation.
And keep you posted here
